src/Module/Settings/Display.php

```diff
@@ -234,11 +234,11 @@ protected function content(array $request = []): string
 
 		$update_content         = $this->pConfig->get($uid, 'system', 'update_content') ?? false;
 		$enable_smile           = !$this->pConfig->get($uid, 'system', 'no_smilies', false);
-		$infinite_scroll        = $this->pConfig->get($uid, 'system', 'infinite_scroll', false);
+		$infinite_scroll        = $this->pConfig->get($uid, 'system', 'infinite_scroll', true);
```

Comment:
If you change a `pconfig` value to `true`, you have to ensure that the default is also changed where the value is read.

Reply:
Ah, this is a much bigger change than I expected 😭  There are 16 places where `infinite_scroll` is used, although only 2 for `stay_local`.  It'll be tricky for me to prove that it's working in all those cases, and that I didn't miss any.  For each usage, I'd need to check the behaviour for a user who has never changed the setting, for one who already enabled the setting, and one who enabled then disabled the setting.

Looking more at how other config settings are handled, I think the right thing to do is add an admin setting for both of these, and that should be the default for each user.  Does that seem like a good idea?

Reply:
It's a fairly simple change. Just check for all occurrences and set the default parameter. Since you use an existing behaviour, I don't think that you need much testing here.

I'm against admin settings in this case. I think that the users should decide.